docs/tutorials/structure_function_showcase.ipynb

```diff
@@ -10,33 +10,85 @@
   },
```

Comment:
<p>nit: a few instances where measurments -&gt; measurements is neded</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I would introduce time lag here, something like Δt, and use it in the leftmost part of the equation, like SF²(Δt), and in the text. Could you give a reference to this relation, I think I understand why we have these -σ here, but it would be nice to have a citation</p><p> </p>